library/src/main/java/com/alamkanak/weekview/DateTimeInterpreter.java

```diff
@@ -7,5 +7,5 @@
  */
 public interface DateTimeInterpreter {
     String interpretDate(Calendar date);
-    String interpretTime(int hour);
+    String interpretTime(int hour, int minutes);
```

Comment:
Should I create a new method with interpretTime(int hour, minutes) and leave the original interpretTime(int hour) to not break people?